README.rst

```diff
@@ -86,12 +86,17 @@ To enable it, configure the port in the config file. For example::
     [http]
     internalapi.http.port = 8091
 
-To use it::
+To use bind::
 
     curl -XPOST 'http://localhost:8091/_matrix/identity/internal/bind' -H "Content-Type: application/json" -d '{"address": "matthew@arasphere.net", "medium": "email", "mxid": "@matthew:matrix.org"}'
 
 The response has the same format as ``/_matrix/identity/api/v1/3pid/bind``.
 
+To use unbind::
+
+    curl -XPOST 'http://localhost:8091/_matrix/identity/internal/unbind' -H "Content-Type: application/json" -d '{"address": "matthew@arasphere.net", "medium": "email", "mxid": "@matthew:matrix.org"}'
+
+The response has the same format as ``/_matrix/identity/api/v1/3pid/unbind``.
```

Comment:
```suggestion
The response has the same format as
`/_matrix/identity/api/v1/3pid/unbind <https://matrix.org/docs/spec/identity_service/r0.3.0#deprecated-post-matrix-identity-api-v1-3pid-unbind>`_.
```

Linking is useful so that someone can quickly get the response format definition.

Changing the link to `/bind` above as well would be useful while you're here :)